admin/index.js

```diff
@@ -1,11 +1,11 @@
 module.exports = AccountAdmin
 
-var getSession = require('../helpers/get-session')
+var getSession = require('../utils/get-session')
```

Comment:
I know we don’t call it "utils" in all repos, but in most, let’s try to settle on it :)